app/javascript/pages/country/embed/embed-component.js

```diff
@@ -0,0 +1,35 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+import Widget from 'pages/country/widget';
+import Share from 'components/share';
+import CountryDataProvider from 'pages/country/providers/country-data-provider';
+
+import WIDGETS_CONFIG from 'pages/country/data/widgets-config.json';
+
+class Embed extends PureComponent {
+  render() {
+    const { widgetKey } = this.props;
+
+    return (
+      <div className="row">
+        <div
+          className={`large-${WIDGETS_CONFIG[widgetKey].gridWidth} small-12`}
```

Comment:
I am not sure about including the grid inside the embed. It doesnt make sense to me. The widgets might need to be responsive? If so we need responsive iframes. Therefore we need to address all the styles and loading.

Reply:
Yeah, I don't like it either, but without that the styles doesn't work properly, so I decided to continue with the grid and later try to find a solution without the grid.
About the responsive, the widgets are 100% responsives.

Reply:
I cant seem to get them to be responsive when embedding. They dont fill the width of the container they are embedded inside of.